tests/integration/test_ols.py

```diff
@@ -49,7 +50,9 @@ def test_status() -> None:
 
 # the raw prompt should just return stuff from LLMChain, so mock that base method in ols.py
 @patch("ols.app.endpoints.ols.LLMChain", new=mock_llm_chain("test response"))
-def test_raw_prompt(monkeypatch) -> None:
+# during LLM init, exceptins will occur on CI, so let's mock that too
+@patch("ols.app.endpoints.ols.LLMLoader", new=mock_llm_loader(None))
```

Comment:
i don't understand how this is working....how does the mock llm loader end up ensuring that the raw prompt request returns "test response" as the result?

Reply:
well we just need to skip the `LLMLoader.__init__` failure, because it is initialized in the code, but unused (because the `LLMChain` is mocked too).

Reply:
i missed the existing patch command for mock_llm_chain, my bad